Packs/SymantecDLP/Integrations/SymantecDLPV2/SymantecDLPV2.py

```diff
@@ -0,0 +1,909 @@
+
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            "Symantec DLP incidents status",
+            camelize(incidents_status_result),
+            removeNull=True,
+        ),
+        outputs_prefix='SymantecDLP.IncidentStatus',
+        outputs_key_field='id',
+        outputs=incidents_status_result,
+    )
+
+
+def get_incident_history_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    incident_id = arg_to_number(args.get('incident_id'))
+    limit = arg_to_number(args.get('limit', 50))
+    incident_history_result = client.get_incident_history_request(incident_id)
+    incident_history_result = incident_history_result[:limit]
+    incidents_history_hr = get_readable_output_incident_history(incident_history_result)
+
+    incidents_history_context = get_context_incident_history(incident_history_result)
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            f"Symantec DLP Incident {incident_id} history results",
+            incidents_history_hr,
+            removeNull=True
+        ),
+        outputs_prefix='SymantecDLP.IncidentHistory',
+        outputs_key_field='incidentId',
+        outputs=remove_empty_elements(incidents_history_context),
+    )
+
+
+def get_list_remediation_status(client: Client) -> CommandResults:
+    remediation_status_result = client.get_list_remediation_status_request()
+    remediation_status_output = get_hr_context_incidents_status(remediation_status_result)
+
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            "Incidents remediation status results",
+            camelize(remediation_status_output),
+            removeNull=True
+        ),
+        outputs_prefix='SymantecDLP.IncidentRemediationStatus',
+        outputs_key_field='id',
+        outputs=remediation_status_output,
+    )
+
+
+def is_incident_already_fetched_in_previous_fetch(last_update_time, incident_creation_time, last_incident_id,
+                                                  incident_id):
+    """
+    Checks if the incident was already fetched
+    :param last_update_time: last_update_time from last_run
+    :param incident_creation_time: The current incident creation time
+    :param last_incident_id: last_incident_id from last_run
+    :param incident_id: The current incident id
+
+    """
+    if (last_update_time and last_update_time > incident_creation_time) or \
+            (last_incident_id and int(last_incident_id) >= int(incident_id)):
+        return True
+    return False
+
+
+def fetch_incidents(client: Client, fetch_time: str, fetch_limit: int, last_run: dict, incident_types: List[str] = None,
+                    incident_status_id: List[str] = None, incident_severities: List[str] = None):
+    """
+    Performs the fetch incidents functionality, which means that every minute if fetches incidents
+    from Symantec DLP and uploads them to Cortex XSOAR server.
```

Comment:
Add a more detailed explanation about how the fetch works and why.
For example, why we might have already fetched incidents found more than once.
Non trivial knowledge that can't be reflected in the code should be recorded in the docstring